geonode/base/api/views.py

```diff
@@ -209,19 +217,26 @@ def resource_types(self, request):
         resource_types = []
         for _model in apps.get_models():
             if _model.__name__ == "ResourceBase":
-                resource_types = [_m.__name__.lower() for _m in _model.__subclasses__()]
-        if "geoapp" in resource_types:
-            resource_types.remove("geoapp")
-        if "service" in resource_types:
-            resource_types.remove("service")
+                resource_types = [{
+                    "name": _m.__name__.lower(),
+                    "count": _m.objects.filter(resource_type=_m.__name__.lower()).count()
```

Comment:
Count must be based on current user @marthamareal

Reply:
This will also need changes [here](https://github.com/GeoNode/geonode/pull/7483/files#diff-7a787998e5e7343bea646b19948d0eee47b1f82758b071696fd0767568782b27R147)